src/net.h

```diff
@@ -49,6 +49,11 @@ class CScheduler;
 class CNode;
 class BanMan;
 
+/** Default for -whitelistrelay. */
+static const bool DEFAULT_WHITELISTRELAY = true;
+/** Default for -whitelistforcerelay. */
+static const bool DEFAULT_WHITELISTFORCERELAY = false;
```

Comment:
16631: was `true` in our code and original PR was not changing it, this was changed in 15193